src/Git.hs

```diff
@@ -367,7 +369,13 @@ runGit userConfig repoDir operation =
       -- not exist.
       gitResult <- callGitInRepo ["push", "--force", "origin", refSpec (sha, branch)]
       case gitResult of
-        Right _ -> pure $ cont PushOk
+        Right _ -> do
+          let delayMicroseconds = 2 * 1_000_000
+          -- TODO: Find a safer way to make sure Github doesn't get confused
+          -- by 2 pushes close together, the delay is all arbitrary and not nice.
+          -- See https://github.com/channable/hoff/issues/196
+          liftIO $ threadDelay delayMicroseconds
+          pure $ cont PushOk
```

Comment:
[The location where this change was requested is not in the `runGit` executor but in the definition of a promotion. We have two `TryPromote*`  functions:

workaround-erroneous-closed-pr](https://github.com/channable/hoff/blob/ccf1ba37e9bbf10f5b8de10bb56e286ffb28bdcc/src/Logic.hs#L243-L250)

https://github.com/channable/hoff/blob/ccf1ba37e9bbf10f5b8de10bb56e286ffb28bdcc/src/Logic.hs#L252-L261

We should add the delay to those flows instead.

---

Additional note when you are updating `TryPromoteWithTag`:
We ignore the `forcePush`'s result in that flow we should bail out there as well.